ui/components/app/snaps/insight-warnings/insight-warnings.js

```diff
@@ -124,10 +125,24 @@ export default function InsightWarnings({
         <Text variant={TextVariant.bodyMd} paddingBottom={4}>
           {warnings.length === 1
             ? t('insightWarningContentSingular', [
+                <Text
+                  as="span"
+                  key={warnings.id}
+                  fontWeight={FontWeight.Medium}
+                >
+                  {`${warnings.length} ${t('insightWarning')}`}
```

Comment:
`warning.length` should be passed into the translation variables so that translators have more context when translating

Reply:
My bad, you're right !